ui/stories/charts/line-chart.stories.js

```diff
@@ -153,6 +153,7 @@ export let Annotations = () => {
       <div class="block" style="height:250px">
         {{#if (and this.data this.annotations)}}
           <LineChart
+            class="with-annotations"
```

Comment:
This isn’t the right place in the file but it looks like something needs to be changed for the [“Live Data” story](https://nomad-storybook-and-ui-j19zf6gjo-hashicorp.vercel.app/storybook/?path=/story/charts-line-chart--live-data) in Storybook:

<img width="1450" alt="image" src="https://user-images.githubusercontent.com/43280/108865528-f46abc00-75b8-11eb-831f-856571cb8ff2.png">

Reply:
Ooooooof. Yeah, I had to change the way `xFormat` works and it looks like that story provides a custom one. Will fix.